### PR TITLE
Modify cfgs related to virtio-fs since s390x supported

### PR DESCRIPTION
**Pull request by fbq815**
delete "no s390x" in these cfgs and modify numa related info to other
architecture

ID:1915754

Signed-off-by: Boqiao Fu <bfu@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/virtio_fs_multi_vms.cfg [hunk 1]
```diff
@@ -28,15 +27,16 @@
...
-    guest_numa_nodes_vm1 = shm0
-    guest_numa_nodes_vm2 = shn0
-    guest_numa_nodes_vm3 = sho0
-    numa_memdev_shm0 = mem-mem1
-    numa_memdev_shn0 = mem-mem1
-    numa_memdev_sho0 = mem-mem1
-    numa_nodeid_shm0 = 0
-    numa_nodeid_shn0 = 0
-    numa_nodeid_sho0 = 0
+    i386, i686, x86_64, ppc64, ppc64le, aarch64:
```
Comment (PaulYuuu) [inline qemu/tests/cfg/virtio_fs_multi_vms.cfg:30]: !s390, s390x:

Comment (fbq815) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg]: finish editing

**Comment by fbq815** [comment]
Test result:
JOB ID     : f9fa0f9e4e6afbd00e8d5b351472ad4c888009d9
JOB LOG    : /root/avocado/job-results/job-2021-01-14T07.25-f9fa0f9/job.log
 (1/1) Host_RHEL.m8.u4.product_av.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.virtio_fs_readonly.s390-virtio: ERROR: No ipv4 DHCP lease for MAC 9a:5b:6b:24:bb:f3 (60.88 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-01-14T07.25-f9fa0f9/results.html
JOB TIME   : 61.80 s
@PaulYuuu  please help me check this patch, this only delete with no 390x, not fully supported yet

File: qemu/tests/cfg/virtio_fs_multi_vms.cfg [hunk 2]
```diff
@@ -1,5 +1,4 @@
 - virtio_fs_multi_vms:
-    no s390x
```
Comment (PaulYuuu) [inline qemu/tests/cfg/virtio_fs_multi_vms.cfg:2]: s390x:
    required_qemu = [5.2.0, )

Comment (fbq815) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg:2]: finish editing

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**